advanced/enable_extensions.md

```diff
@@ -28,3 +28,17 @@ activation commands in your user enviroment script.
 [nbextensions]: ../images/nbextensions.png "Nbextensions configuration panel"
 [terminal_ext]: ../images/terminal_ext.png "Extension activation on terminal"
 [spellchecker_nb]: ../images/spellchecker_nb.png "Extension activation inside a notebook"
+
+
+# Enable Jupyter extension not provided by SWAN.
+
+Install Jupyter extension requires to run the next commands from a SWAN terminal:
+
+`pip install --user extension_name`
```

Comment:
This individual lines with `...` are not showing correctly.
Can you create just one block with
```shell
line1
line2
...
```